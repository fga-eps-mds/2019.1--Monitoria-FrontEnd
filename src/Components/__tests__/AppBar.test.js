import React from 'react';
import AppBar from '../AppBar/AppBar';
import { shallow } from 'enzyme';

describe('Testing AppBar component', () => {
    it('Test if AppBar renders correctly', () =>{
        const tree = shallow(<AppBar/>);
        expect(tree).toMatchSnapshot();
    });
});