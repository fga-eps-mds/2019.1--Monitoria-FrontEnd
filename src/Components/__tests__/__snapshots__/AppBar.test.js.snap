// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing AppBar component Test if AppBar renders correctly 1`] = `
<SimpleAppBar
  classes={
    Object {
      "root": "SimpleAppBar-root-1",
    }
  }
/>
`;
